src/components/Events.module.scss

```diff
@@ -1,8 +1,26 @@
-.container {
+.event {
+  align-items: left;
+  height: 200px;
+  width: 343px;
+  border: 2px solid #333;
+  padding-top: 5px;
+  padding-left: 10px;
+  border-radius: 10px;
+}
+
+.list {
   display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  height: 100%;
+  flex-wrap: wrap;
   width: 100%;
+  gap: 30px;
+  justify-content: space-between;
 }
+
+$grid-breakpoints: (
```

Comment:
Usually, you create a file and import it from there. Check profiles - https://github.com/monster-next/profiles-single-page-application/blob/master/src/common/styles/breakpoint.scss

and import this like this https://github.com/monster-next/profiles-single-page-application/blob/master/src/common/components/alertsAndPreferrences/TagOptionList/TagOptionList.module.scss

Reply:
done